### PR TITLE
Port controllers, routes, etc. to replicate existing functionality via package

### PR DESCRIPTION
**Pull request by kylebarney**

### CODE REVIEW

File: src/ToolServiceProvider.php [hunk 1]
```diff
@@ -3,6 +3,10 @@
 namespace Tipoff\StaffBookings;
 
 use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\Facades\Route;
+use Laravel\Nova\Events\ServingNova;
+use Laravel\Nova\Nova;
+use Tger\Booking\Http\Middleware\Authorize;
 
 class ToolServiceProvider extends ServiceProvider
```
Comment (sl0wik) [inline src/ToolServiceProvider.php:11]: I guess namespace should fit the package name.

Comment (kylebarney) [reply src/ToolServiceProvider.php:11]: Good catch - missed that one. I've fixed and tested that issue and pushed up the fix commit to this branch.

**Comment by drewroberts** [comment]
All of these routes and controllers will be in the tipoff/bookings package. This package is just a layer on top of it and should require that package.

**Comment by drewroberts** [comment]
I deleted out the files that I think should live in the other packages. This package will require those packages. It may have some different controllers that live here and are exclusive for staff bookings. Let's get into that in a Zoom meeting next week. Thank you!

**Review by drewroberts — APPROVED** [review]
I'm going to push this PR on through. You can make any adjustments you need in a new PR. Thank you! 🏂

**Pull request merged**